ddtrace/utils/http.py

```diff
@@ -7,3 +7,15 @@ def normalize_header_name(header_name):
     :rtype: str
     """
     return header_name.strip().lower() if header_name is not None else None
+
+
+def strip_query_string(url):
```

Comment:
we have to have a _ton_ of example tests for this.

just make sure we dot t's and cross i's on writing our own sanitizer.